### PR TITLE
Ci/travis

### PR DESCRIPTION
**Pull request by totorototo**
move from bitrise to travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  licenses:
+  - android-sdk-preview-license-.+
+  - android-sdk-license-.+
+  - intel-android-extra-license.+
+script:
+- cd android && ./gradlew assembleRelease
+deploy:
+  provider: releases
+  api_key:
+    secure: okpBUibQ+A+4CDvgUlMIq8kw1w9MfmjKLe2RHCAFfwVFHlCKgdDaw2rCXgkP4wcpBkcQqH0WiP0/PqeF/o+Y8K5Co6GRbaPBnCkIFQXWM8fqrICFK0fPY3K2dXUTgYXbsM5bp8P2IC38trhuWNKiOlXCsQL7Q/DVqUdpvc3pmqYAliavOWQHj8xzhnq8otzIjknWiqt7f1f7p68kAD33j8ZoCpdwz/mIAhIYcKT3VibNscJYBFxvdbCIKyH392ai5bXk3ThQVeKAEQtnqW93tM7fXfYOHLQdrKpsGz02mgw0E64i/ulN4wSmNEpWnPqED+WaxoBfYc3cAylvLh0Eom32BZmnSvJTTKSHJYNtxAXbSxvgAAZz4qAcjHccDfu2wQuJ3j1rfaGjtG4kro62tG8jpIHgc6VAPlBXNF9rsfMn5XHaBvTe/mChOD+EY6bvBix+24V5i09CbXCZwxf+cpWqdse5nkaL8GmFqmWX0bLAQHj6DGy1Z1MCTIeZTbpVu5ocbFeOXHwPEHbm2Fb55qyx8YmvPvCOtqRE7nqe5JYPWpPraMJUfg0s32Vo8/kBzU6Hkm5hPmOMSPQDrNdMzin+4Zej1/nkWNdAdFgkXua8sG4C3e4ltx2YxWFNAfrhxlc+9R04Ja8RUns8m/0zbQXThEmZhVkxYufb0xq/tq8=
```
Comment (guipasmoi) [inline .travis.yml]: are u sure sure travis.yml should be inside the repo??
is there any sensible data? (api_key,...)?
 
if that is the case, you can forget this PR and revoke all sensible data.

Comment (guipasmoi) [reply .travis.yml]: just to be sure

File: set_tags.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    if [ -z "$TRAVIS_TAG" ]; then
+      echo -e "Starting to tag commit.\n"
+
+      git config --global user.email "travis@travis-ci.org"
+      git config --global user.name "Travis"
+
+      # Add tag and push to master.
+      git tag -a v${TRAVIS_BUILD_NUMBER} -m "Travis build $TRAVIS_BUILD_NUMBER pushed a tag."
+      git push origin --tags
+      git fetch origin
```
Comment (guipasmoi) [inline set_tags.sh]: when is that triggered?

File: .gitignore [hunk 3]
```diff
@@ -52,3 +52,6 @@ fastlane/Preview.html
 fastlane/screenshots
 /.tmp/
 /.env
+*.keystore
+*.tar
```
Comment (guipasmoi) [inline .gitignore:56]: what is in *.tar?

File: .travis.yml (hunk 1 above)
Comment (totorototo) [reply .travis.yml]: 1- it has to be,
2- all secured keys or critic data is encrypted, no worries (cf travis documentation).
3- it all just fine! 👍 

all secured and generated by travis command line tool!

File: set_tags.sh (hunk 2 above)
Comment (totorototo) [reply set_tags.sh]: 1- for a given branch,
2- not a pull request,
3- not triggered by a tag!

File: .gitignore (hunk 3 above)
Comment (totorototo) [reply .gitignore:56]: all secured data (key + env+ ...) are compressed before being encrypted using travis command line tool. this is just to be not to commit non encrypted data! 

https://fr.wikipedia.org/wiki/Tar_(informatique)
📦

File: .travis.yml (hunk 1 above)
Comment (guipasmoi) [reply .travis.yml]: image

File: set_tags.sh (hunk 2 above)
Comment (guipasmoi) [reply set_tags.sh]: i know how to read but that doesnt help

File: .gitignore (hunk 3 above)
Comment (guipasmoi) [reply .gitignore:56]: ok

**Comment by guipasmoi** [comment]
is it possible to move travis file inside a folder travis ? to isolate them

File: set_tags.sh (hunk 2 above)
Comment (totorototo) [reply set_tags.sh]: oops!
you don't us to add tag on release?

**Comment by totorototo** [comment]
travis.yml file has to be where it is now (project root) and there is nothing we could do!
scripts could be moved in scripts directory if needed

**Review by guipasmoi — APPROVED** [review]

**Pull request merged**